test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj

```diff
@@ -1,8 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <Import Project="..\..\build\common.props" />
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="$(MicrosoftAspNetCoreAllPackageVersion)" />
```

Comment:
This should be equivalent to 2.0.0. These are regression tests on the latest version of ANCM.

Reply:
bump on this. This should be 2.0.0 equivalent.